velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -75,13 +75,42 @@ static std::vector<std::string> kRangeFrameClauses = {
 
 /// Common set of window function frame clauses in ROWS mode, with current row,
 /// unbounded preceding, and unbounded following frame combinations.
-static std::vector<std::string> kRowsFrameClauses = {
+static std::vector<std::string> kUnboundedRowsFrameClauses = {
     "rows unbounded preceding",
     "rows current row",
     "rows between current row and unbounded following",
     "rows between unbounded preceding and unbounded following",
 };
 
+/// Common set of window function frame clauses in ROWS mode, with current row,
```

Comment:
You can remove the listing of the frame types in the comment (just mention that they have k Preceding and Following) as they are obvious in the listing below.

Reply:
Updated, thanks!